src/sass/abstract/_flex.scss

```
@@ -0,0 +1,14 @@
+@mixin flex($justify: flex-start, $align: stretch, $direction: row, $wrap: nowrap, $gap: 0) {
+    display: flex;
+    justify-content: $justify;
+    align-items: $align;
+    @if $direction != 'row' {
+        flex-direction: $direction;
+    }
+    @if $wrap != 'nowrap' {
+        flex-wrap: $wrap;
+    }
+    @if $gap != 0 {
+        gap: $gap;
+    }
```

```suggestion
  flex-direction: $direction;
  flex-wrap: $wrap;
  gap: $gap;
```
khi dùng chỉ cần truyền `direction` khi `direction != row`, nếu tham số `direction = row` thì không cần truyền tham số `direction`